docs/_posts/josejuanmartinez/2022-01-17-ner_deid_subentity_roberta_es.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: model
+title: Detect PHI for Deidentification purposes (Spanish)
+author: John Snow Labs
+name: ner_deid_subentity_roberta
+date: 2022-01-17
+tags: [deid, es, licensed]
```

Comment:
don't we usually add the `clinical` tag here?

Reply:
`clinical` tag only applies to the models trained with clinical glove embeddings

Reply:
oh, good to know, I will share with the others